cmd/ccboc/main.go

```diff
@@ -121,6 +121,58 @@ func getCalculationID(c *cli.Context) error {
 	return nil
 }
 
+func getCalculationResult(c *cli.Context) error {
+	teff := c.Float64("teff")
+	logG := c.Float64("logG")
+
+	fmt.Println(globalConfig.APIURL + "/calculations/results?teff=" + fmt.Sprintf("%0.1f", teff) + "&logg=" + fmt.Sprintf("%0.1f", logG))
+
+	body, responseError, err := request("GET", globalConfig.APIURL+"/calculations/results?teff="+fmt.Sprintf("%0.1f", teff)+"&logg="+fmt.Sprintf("%0.1f", logG), bytes.NewBuffer(nil))
+	if err != nil {
+		logrus.WithError(err).Fatal("error while perfoming the http request")
+	} else if responseError != nil {
+		logrus.WithFields(logrus.Fields{"message": responseError.Message, "status_code": responseError.StatusCode}).Fatal("errors occurred")
+	}
+
+	fileName := fmt.Sprintf("%0.1f___%0.1f", teff, logG)
+
+	file, err := os.Create(fileName)
```

Comment:
First, here we should check the error. Second, this will create the file in the current directory. We should have a default location for the downloaded results. Maybe in the same path where we create the configs for this tool?

Reply:
Somehow, I am unable to create the file in another directory. Tried all sorts of functions, different permission..
os.Chdir is not doing the same thing as terminal for me. Couldnt find anything that would help me solve this.

Reply:
This should be related to your environment only. I guess permissions? You should be able to write in the `~/.config/ccboc` since we already wrote the configuration files.